IfcPlusPlus/CMakeLists.txt

```diff
@@ -75,9 +74,14 @@ if (MSVC)
 	set_source_files_properties(src/ifcpp/IFC4X3/TypeFactory.cpp PROPERTIES COMPILE_FLAGS /bigobj)
 endif()
 
-add_library(IfcPlusPlus STATIC ${IFCPP_SOURCE_FILES})
+add_library(IfcPlusPlus ${IFCPP_SOURCE_FILES})

```

Comment:
Default is STATIC unless `BUILD_SHARED_LIBS` is `ON` - no change of behavior here.